### DISCUSSION TITLE
Got so many XHR error

### DISCUSSION THREAD

**Discussion by dels07** [algolia/instantsearch#4623] [Q&A]
Hi there,

I setup log monitor using datadog and got so many errors like this

```
XHR error POST https://9vc7ymikz2-dsn.algolia.net/1/indexes/*/queries?x-algolia-agent=Algolia%20for%20JavaScript%20(4.8.3)%3B%20Browser%3B%20instantsearch.js%20(4.10.0)%3B%20JS%20Helper%20(3.3.4)
```

Checking for the error it mentioned:

```
error {
  origin: 'network'
}
```

Any idea what is this and possibility to fix or filter out this kind of errors?

**Comment by Haroenv** [comment]
The Algolia JavaScript client actually already catches those errors, and falls back from -dsn to -3, -2 and -1 domains. Is this something you see in certain real situations or only in logs?

**Reply by dels07** [reply]
Currently only see it via log in datadog dashboard, it's floading :D so I hope I can remove this error since datadog don't have error filtering before send to their server.

![Screenshot from 2021-01-13 10-45-28](https://user-images.githubusercontent.com/4617477/104404104-7e703f80-558c-11eb-83b1-4aa3c17c5a04.png)

**Reply by Haroenv** [reply]
I think this is more relevant for https://github.com/DataDog/browser-sdk/issues/308

**Reply by dels07** [reply]
I see, thanks, I'll watching that issue
